master/getting-started/kubernetes/index.md

```diff
@@ -1,41 +1,195 @@
-includes links to a number of popular guides and installers which use Calico. It also
-includes information on installing Calico on a from-scratch Kubernetes cluster using either a self-hosted Kubernetes manifest,
-or by integrating Calico into your own configuration management scripts.
+To deploy a cluster suitable for production, refer to [Installation](/getting-started/kubernetes/installation/).
 
-## Using Calico with Kubernetes
 
-Once you have a Kubernetes cluster with Calico installed, the following articles will help you
-get familiar with Calico and make the most of the features that Calico provides.
+### Requirements
 
-##### Tutorials
+- AMD64 processor
+- 2CPU
+- 2GB RAM
+- 15GB free disk space
+- RedHat Enterprise Linux 7.x+, CentOS 7.x+, Ubuntu 16.04+, or Debian 8.x+
 
-**[Using the NetworkPolicy API](tutorials/simple-policy)**: this guide explains how to use Calico to secure a simple two-tier application
-using the Kubernetes NetworkPolicy API.
 
-**[Advanced Calico Policy](tutorials/advanced-policy)**: this guide explains how to use Calico to provide policy features beyond
-what can be done with the Kubernetes NetworkPolicy API like egress and CIDR based policy.
+### Before you begin
 
-**[Stars Demo](tutorials/stars-policy/)**: this demo features a UI which actively shows blocked and allowed connections as policy is implemented.
+[Follow the Kubernetes instructions to install kubeadm](https://kubernetes.io/docs/setup/independent/install-kubeadm/){:target="_blank"}.
 
-##### Usage Reference
+> **Note**: After installing kubeadm, do not power down or restart
+the host. Instead, continue directly to the 
+[next section to create your cluster](#create-a-cluster).
+{: .alert .alert-info}
 
-**[Using the calicoctl CLI tool][calicoctl]**: reference documentation for the Calico CLI tool, calicoctl.
 
-**[Configuring BGP Peering][bgp-peering]**: this guide is for users on private cloud who want to configure Calico to peer with their underlying infrastructure.
+### Create a single-host Kubernetes cluster
 
-[calicoctl]: {{site.baseurl}}/{{page.version}}/reference/calicoctl/
-[bgp-peering]: {{site.baseurl}}/{{page.version}}/usage/configuration/bgp
+1. As a regular user with sudo privileges, open a terminal on the host that 
+   you installed kubeadm on. 
+
+1. Update your package definitions and upgrade your existing packages.
+
+   ```
+   sudo apt-get update && sudo apt-get upgrade
+   ```
+   
+1. Initialize the master using the following command.
+
+   ```
+   sudo kubeadm init --pod-network-cidr=192.168.0.0/16 --skip-preflight-checks
+   ```
+   
+1. Execute the following commands to configure kubectl (also returned by
+   `kubeadm init`).
+
+   ```
+   mkdir -p $HOME/.kube
+   sudo cp -i /etc/kubernetes/admin.conf $HOME/.kube/config
+   sudo chown $(id -u):$(id -g) $HOME/.kube/config
+   ```
+   
+1. Install Calico, a pod network, and a single node etcd with the 
```

Comment:
I think this is misleading - Calico _is_ a pod network.

So it should probably be "Install Calico and a single node etcd" or, if you think the "pod network" bit is important, it should just say "Install Calico (a pod network) and a single node etcd"